src/lib/browser/BrowserPage.ts

```
@@ -668,6 +670,224 @@ export class BrowserPage {
+  // Deprecated: background pdf.js extraction removed in favor of unified side-panel pdf.js path
+  async getPdfText(_options?: { maxPages?: number }): Promise<string> {
+    throw new Error('Background PDF extraction is disabled. Use side panel PDF parsing only.')
+  }
+
+  /**
+   * Get total number of pages in the current PDF
+   */
+  async getPdfPageCount(): Promise<number> {
+    const sourceUrl = await this._resolvePdfSourceUrl();
+    const bytes = await this._fetchPdfBytes(sourceUrl);
+    try { (GlobalWorkerOptions as any).workerSrc = chrome.runtime.getURL('pdf.worker.mjs') } catch {}
```

ESLint

[error] Empty block statement.

(no-empty)